README.md

```diff
@@ -30,7 +30,13 @@
 
 **Location:** GitHub Issues [(see all accepted proposals).](https://github.com/withastro/roadmap/issues)
 
-**What to Expect:** A proposal reaches this stage (aka "is accepted") during an [#rfc-meetings](RFC Meeting) with Maintainers and TSC (>= L2), following our existing [RFC voting](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) process.
+**What to Expect:**
+
+RFCs advance through the stages during RFC meetings with TSC and maintainers. Voting follows the RFC proposal voting process.
```

Comment:
Maybe we should use a bulleted list to make it clearer those 3 paragraphs are for "What to Expect"